src/main/resources/static/style.css

```diff
@@ -52,18 +55,21 @@ h3{
 	font-size: 40px;	
 }
 
-
-
 .container {
   width: 80%;
+  height: 200px;
+  padding: 10px;
+  float: left;
 }
 
 .one {
```

Comment:
While you are here, these could use more descriptive names than "one" and "two".